etc/enigma.appdata.xml

```diff
@@ -44,22 +44,47 @@
         <ul>
           <li>Rescaling the application window introduces graphical artifacts: Better use fixed window sizes for now.</li>
           <li>Not all fullscreen modes are available yet, in particular no vertical screen modes. If you use a vertical screen, please use Enigma in windowed mode.</li>
-          <li>On Windows 7, after playing or finishing a level, the mouse cursor is trapped inside Enigma's window (but not frozen). Enigma can still be operated and exited via the main menu or the game menu, though.</li>
+          <li>On Windows 7, after playing or finishing a level, the mouse cursor is trapped inside Enigma&apos;s window (but not frozen). Enigma can still be operated and exited via the main menu or the game menu, though.</li>
         </ul>
       </description>
     </release>
   </releases>
-  <url type="homepage">http://www.nongnu.org/enigma/</url>
+  <url type="homepage">https://www.nongnu.org/enigma/</url>
   <screenshots>
-    <screenshot type="default">http://www.nongnu.org/enigma/images/screenshots/ral15_1.png</screenshot>
-    <screenshot>http://www.nongnu.org/enigma/images/screenshots/mecke22_2.png</screenshot>
-    <screenshot>http://www.nongnu.org/enigma/images/screenshots/luc36_1.png</screenshot>
+    <screenshot type="default">
+      <image>https://www.nongnu.org/enigma/images/screenshots/ral15_1.png</image>
+      <caption>"The Gordian Knot" by Ronald Lamprecht</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://www.nongnu.org/enigma/images/screenshots/mecke22_2.png</image>
+      <caption>Example Level</caption>
+    </screenshot>
+    <screenshot>
+      <image>https://www.nongnu.org/enigma/images/screenshots/luc36_1.png</image>
+      <caption>Example Level</caption>
+    </screenshot>
   </screenshots>
   <keywords>
     <keyword>enigma</keyword>
     <keyword>puzzle</keyword>
     <keyword>marble</keyword>
   </keywords>
-  <content_rating type="oars-1.1" />
-  <updatecontact>enigma-devel@nongnu.org</updatecontact>
-</application>
+  <developer id="enigma.nongnu.org">
```

Comment:
I don't really like to duplicate the list of contributors in yet another place.
But if this is good practice for the metadata, then I'm ok with it. Just make sure it is in sync with what is shown in-game. (Which is the most up-to-date list a think.)

Reply:
Hmm, spec seems to say we can have just one name tag (per language):
https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#developer-name

Although the file validates via `appstreamcli validate`.